### ISSUE TITLE
Django 3.2b1 fails to runserver grappelli.dashboard

### ISSUE THREAD

**Issue by bob-r** [sehmaschine/django-grappelli#957]
with `django.core.exceptions.ImproperlyConfigured: The app label 'grappelli.dashboard' is not a valid Python identifier`.
because of:
- https://github.com/sehmaschine/django-grappelli/blame/master/grappelli/dashboard/apps.py#L9
- https://github.com/django/django/blob/main/django/apps/config.py#L37

**Comment by sehmaschine** [comment]
Thanks. We'll check that once we work on compatibility with Django 3.2.

**Comment by Goldziher** [comment]
I can confirm this is still occuring with the official 3.2

**Comment by Goldziher** [comment]
Any chance for a hotfix for this? the fix is minor - its just changing the label from grappelli.dashboard to grappelli_dashboard. I can add a PR for this in a few minutes. 

This is blocking upgrade to django 3.2 for us, and i guess pretty much everyone else who uses this library's dashboard.

**Comment by voarsh2** [comment]
@Goldziher any clues how I can utilise your hotfix?

**Comment by Goldziher** [comment]
@voarsh2 , you can install the package from my fork?

**Comment by Goldziher** [comment]
@sehmaschine @parsch ?

**Comment by sehmaschine** [comment]
We'll work on compatibility with Django 3.2 later this week.

**Comment by sehmaschine** [comment]
fixed in stable/2.15.x ... you are able to check this branch and test with Django 3.2 for the next couple of days. we plan to do a release early next week (if there are no bigger issues). with my tests, everything seems to work fine.

**Comment by jarshwah** [comment]
As a workaround I *think* you could define your own app config in your own project and point to that?

ie:

```
# myapp.appconfigs
from django.apps import AppConfig


class MyDashboardConfig(AppConfig):
    name = "grappelli.dashboard"
    label = "grappelli_dashboard"
    verbose_name = "Grappelli Dashboard"
```

And then in INSTALLED_APPS:

```
INSTALLED_APPS = [
...
"grappelli",
"myapp.appconfigs.MyDashboardConfig",
...
]
```
